src/test/java/uk/gov/pay/products/service/GatewayAccountUpdaterTest.java

```diff
@@ -31,39 +31,17 @@ public void setUp() throws Exception {
     }
 
     @Test
-    public void shouldUpdateServiceName_whenSingleProduct() throws Exception {
+    public void shouldUpdateServiceName() throws Exception {
```

Comment:
I see another test for this. That it the Dao returns false instead of true

Reply:
It's there, you just missed it